Simlar/src/SMLRLinphoneHandler.m

```diff
@@ -478,7 +495,7 @@ - (AVAudioSessionCategoryOptions)generateAudioSessionCategoryOptions
         case SMLRCallStatusEnded:
             return 0;
         case SMLRCallStatusIncomingCall:
-            return AVAudioSessionCategoryOptionMixWithOthers|AVAudioSessionCategoryOptionDuckOthers|AVAudioSessionCategoryOptionAllowBluetooth;
```

Comment:
Good point.
Do mean something like this:

        case SMLRCallStatusIncomingCall:
            return AVAudioSessionCategoryOptionDefaultToSpeaker| AVAudioSessionCategoryOptionDuckOthers| AVAudioSessionCategoryOptionAllowBluetooth;